src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,129 @@
+# Variational Inference using Pathfinder
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
```

Comment:
This feels redundant with [src/cmdstan-guide/pathfinder_config.Rmd](https://github.com/stan-dev/docs/pull/651/files#diff-f498b1eea44ea4ceffd1f8e28e7b1953b2df2f2566f6de47bd499ae561438f14), so I would just remove the earlier description and put a pointer to here instead.